draft-spaghetti-sidrops-rpki-ta-tiebreaker.xml

```diff
@@ -88,14 +88,14 @@ version="3">
         TA operators periodically reissue TA certificates to introduce changes to, for example, modify the set of IP Address Delegations and/or Autonomous System Identifier Delegations included in the <xref target="RFC3779"/> extension(s), the content of the Subject Information Access extension (<xref target="RFC5280" section="4.2.2.2"/>), and the certificate validity period (<xref target="RFC5280" section="4.1.2.5"/>).
       </t>
       <t>
-        Relying Parties periodically download purported current Trust Anchor certificates from remote locations and verify that the key of the self-signed certificate matches the key embedded in its associated Trust Anchor Locator <xref target="RFC8630"/>.
-        This means that Relying Parties have a choice between using a previously retrieved locally cached copy of the TA certificate and the newly-retrieved instance of the TA certificate.
+        Relying Parties periodically download Trust Anchor certificates from well-known, remote locations and verify that the key of the self-signed certificate matches the key embedded in its associated Trust Anchor Locator <xref target="RFC8630"/>. This transfer may happen via an unauthenticated channel, and the certificate is verified by checking that it is signed by the public key in the TAL.
```

Comment:
Every sentence should be on its own line, this makes git review much easier! :-)